### PR TITLE
[docs][yba] Control Plane RBAC with custom roles (MVP)

### PR DESCRIPTION
**Pull request by ddhodge**
Add custom Roles functionality.
DOC-127

@netlify /preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/rbac-platform.md
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/rbac-platform.md:55]:
```suggestion
* **Backup Admin** has access to the backup related tasks, such as the following:
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/rbac-platform.md
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/rbac-platform.md:28]:
```suggestion
* Navigate to **Admin > User Management > Users**, and click **Add User**.
```

File: docs/content/preview/yugabyte-platform/security/authorization-platform.md
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/security/authorization-platform.md:45]: Why do we want to mention about YSQL roles pertaining to YBM in YBA docs?

File: docs/content/preview/yugabyte-platform/security/authorization-platform.md
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/security/authorization-platform.md:28]:
```suggestion
To view the roles in your universe, enter the following command:
```

File: docs/content/preview/yugabyte-platform/security/security-checklist-yp.md
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/security/security-checklist-yp.md:34]:
```suggestion
For information on how to manage roles and users in YugabyteDB Anywhere universes, see [Database authorization](../authorization-platform).
```

**Review by aishwarya24 — COMMENTED** [review]
some comments

**Comment by ddhodge** [comment]
@haikarthikssk @Arpit-yb Can you review or suggest someone to review

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:21]: which determine what are the actions they can perform.

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:74]: Admin > Access Management > Roles

Comment (vpatibandla-yb) [reply docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:74]: With the new RBAC, we can not only create Custom Roles but also scope the customer roles at the universe level.

Please also include the flow to modify the role bindings of the users from the Admin > Access Management > Users > Edit Assigned Roles option for the user.

Comment (vpatibandla-yb) [reply docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:74]: The new RBAC feature which allows custom roles can be turned on by using the runtime flag yb.rbac.use_new_authz. This is an internal Runtime config as of now. I will make the change to PUBLIC once I get the QA approval.

The runtime config can be edited from the UI from the Admin > Advanced > Global Configuration section and the runtime config key yb.rbac.use_new_authz should be set to true for the user to use the new RBAC logic. Once the runtime config is set to true, it cannot be turned off

Comment (vpatibandla-yb) [reply docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:74]: Please write a section on how to assign universes for Custom roles

Comment (vpatibandla-yb) [reply docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:74]: Phase 1 MVP: The resource that we deal with are Universes. Later phases will deal with other kind of resources

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md
Comment (vpatibandla-yb) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:81]: This is the old flow, do we want to include the new RBAC workflows too?

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:21]:
```suggestion
Users are assigned roles, which define the set of actions users can perform.
```

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:81]: We want only the new flow I would have thought

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:83]: Theres another built-in role called "Connect Only".

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:29]: Would it be better to add images here to help guide the user better?

File: docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md
Comment (Sahith02) [inline docs/content/preview/yugabyte-platform/administer-yugabyte-platform/anywhere-rbac.md:122]: @ddhodge @vpatibandla-yb Should we also add here some of the limitations (from our internal design doc) that we have in RBAC phase 1, which will be solved in phase 2?

**Review by Sahith02 — APPROVED** [review]

**Pull request merged**
